source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -945,6 +960,38 @@ void InjaTransformer::transform(Http::RequestOrResponseHeaderMap &header_map,
   } else if (merged_extractors_to_body_) {
     std::string output = json_body.dump();
     maybe_body.emplace(output);
+  } else if (merge_templates_.size() > 0) {
```

Comment:
nit: https://cplusplus.com/reference/vector/vector/empty/
```suggestion
  } else if (!merge_templates_.empty()) {
```